tests/lvmo/test_lvm_clone_bigger_than_disk.py

```diff
@@ -0,0 +1,221 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier1, skipif_lvm_not_installed
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, acceptance
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import LVM
+from ocs_ci.ocs.exceptions import LvSizeWrong
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@acceptance
+@skipif_lvm_not_installed
+@skipif_ocs_version("<4.11")
+class TestLvmCloneBiggerThanDisk(ManageTest):
+    """
+    Test lvm clone bigger than disk
+
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["volume_mode", "volume_binding_mode"],
+        argvalues=[
+            pytest.param(
+                *[constants.VOLUME_MODE_FILESYSTEM, constants.WFFC_VOLUMEBINDINGMODE],
+                marks=pytest.mark.polarion_id("OCS-4383"),
+            ),
+            pytest.param(
+                *[constants.VOLUME_MODE_BLOCK, constants.WFFC_VOLUMEBINDINGMODE],
+                marks=pytest.mark.polarion_id("OCS-4383"),
+            ),
+            pytest.param(
+                *[
+                    constants.VOLUME_MODE_FILESYSTEM,
+                    constants.IMMEDIATE_VOLUMEBINDINGMODE,
+                ],
+                marks=pytest.mark.polarion_id("OCS-4383"),
+            ),
+            pytest.param(
+                *[constants.VOLUME_MODE_BLOCK, constants.IMMEDIATE_VOLUMEBINDINGMODE],
+                marks=pytest.mark.polarion_id("OCS-4383"),
+            ),
+        ],
+    )
+    def test_create_clone_from_pvc_bigger_than_disk(
+        self,
+        volume_mode,
+        volume_binding_mode,
+        project_factory,
+        lvm_storageclass_factory,
+        pvc_clone_factory,
+        pvc_factory,
+        pod_factory,
+    ):
+        """
+        test create delete snapshot
+        .* Check one disk size
+        .* Create PVC bigger than disk size
+        .* Create POD
+        .* Run IO PVC
+        .* Create clone
+        .* Attach pod
+        .* Check LV size
+        .* Run IO
+
+        """
+
+        access_mode = constants.ACCESS_MODE_RWO
+
+        lvm = LVM(fstrim=True, fail_on_thin_pool_not_empty=True)
+        disk1 = lvm.pv_data["pv_list"][0]
```

Comment:
there is no argument called `disk2` so I guess you can rename this one to something like `disk_name`

Reply:
As I'm requesting the first disk in the list `lvm.pv_data["pv_list"][0]` it seems reasonable to call it that way.

Reply:
that is not clear from the arg name

Reply:
pv_data is a parse from pvs command. In that level persistent volume is either a disk or a partition.